git_machete/client.py

```diff
@@ -2168,15 +2168,15 @@ def __get_added_remote_name_or_none(self, github_domain: str, remote_url: str) -
                 return remote
         return None
 
-    def retarget_github_pr(self, head: LocalBranchShortName) -> None:
+    def retarget_github_pr(self, head: LocalBranchShortName, ignore_if_missing: bool) -> None:
         domain = self.__derive_github_domain()
         remote_org_repo = self.__derive_remote_and_github_org_and_repo(domain=domain, branch_used_for_tracking_data=head)
         github_client = GitHubClient(domain=domain, organization=remote_org_repo.organization, repository=remote_org_repo.repository)
 
         debug(f'organization is {remote_org_repo.organization}, repository is {remote_org_repo.repository}')
 
         pr: Optional[GitHubPullRequest] = github_client.derive_pull_request_by_head(head)
-        if not pr:
+        if not pr and not ignore_if_missing:
             raise MacheteException(f'No PR is opened in {bold(remote_org_repo.organization)}/{bold(remote_org_repo.organization)} '
```

Comment:
Even if `ignore_if_missing == True`, the execution must **not** continue when the PR is missing

Reply:
@amalota By not continuing you mean raising the exception or just quietly terminating execution?